### ISSUE TITLE
Usability improvements for setup page

### ISSUE THREAD

**Issue by jbravo94** [senaite/senaite.core#2463]
## Description

Usability improvements for setup page

## Reproduction guide

A: Open Setup Page => Methods Panel is not selectable (Usually available in main page sidebar but not anymore if field is hidden via navigation settings)
B: Open Setup Page => Lab Departments visible => Open e.g. Lab Contacts => Lab Departments not visible in sidebar

## Observed behaviour

A: Methods Panel in Setup page missing
B: Lab Departments in Setup sidebar missing

## Expected behaviour

A: Methods Panel in Setup page visible
B: Lab Departments in Setup sidebar visible

## Versions

- OS: Linux

- senaite.core: 2.4.1

## Screenshot

Lab Departments in sidebar missing:
![image](https://github.com/senaite/senaite.core/assets/26746017/5b3e1a3d-539e-47e7-893c-0838d7414b23)

Methods missing in setup page:
![image](https://github.com/senaite/senaite.core/assets/26746017/cd3ca025-2b03-45f7-b241-d34657a307b0)

**Comment by jbravo94** [comment]
If this issue is acknowledged I can try to fix it on my own.

**Comment by ramonski** [comment]
Hi @jbravo94,

Thanks for sharing your improvement suggestions!

That the we do not have the "Methods" visible in the setup has some historical architectural reasons mainly because of permissions.
Basically it is not located below the "Setup" folder, but is located in the site root.
However, it would make sense to have it the setup view listed, agree with that.

The other issue you mentioned will be probably obsolete in the future, because we are moving to a new Dexterity based setup folder, which won't list the contents in the sidebar.
E.g. it will only allow navigation through the setup tiles view.

In summary, you are welcome to add a static setup tile for the "Methods" in the setup view, but the sidebar changes won't make it in the core for now.

Thanks and best regards
Ramon
